firehose-cloudwatch-trigger-stack.yaml

```diff
@@ -2,7 +2,7 @@ AWSTemplateFormatVersion: '2010-09-09'
 
 Parameters:
   LogGroupConfig:
-    Description:  "String representation of JSON array of objects of your CloudWatch Loggroup(s) and respective filter (if applicable)"
+    Description: "JSON array of objects representing your LogGroup and Filters (if applicable). For example: [{\"LogGroupName\":\"logGroup1\",\"FilterPattern\":\"filter1\"}]"
```

Comment:
Do we need escape character?

Reply:
Yes without escape character linting is failing. Also I have done build and deploy and tested this.